content/applications/websites/livechat/reports.rst

```diff
@@ -0,0 +1,176 @@
+=======
+Reports
+=======
+
+Odoo *Live Chat* includes several reports that allow for the monitoring of operator performance and
+the identification of trends in customer conversations.
+
+Available reports
+=================
+
+The following reports are included in the *Live Chat* app:
+
+- :ref:`Sessions History <livechat/sessions-history>`
+- :ref:`Session Statistics <livechat/session-statistics>`
+- :ref:`Operator Analysis <livechat/operator-analysis>`
+
+.. note::
+   The *Live Chat Ratings Report* can also be accessed through the :guilabel:`Report` menu. For more
+   information on this report, and on the *Live Chat* rating process, see :doc:`Live Chat Ratings
+   <../livechat/ratings>`.
+
+To access the available reports, navigate to the :menuselection:`Live Chat app --> Report menu`.
+
+.. _livechat/sessions-history:
+
+Sessions History
+----------------
+
+The *Sessions History* report displays an overview of live chat sessions, including participant
+names, session dates, and the duration of the session. It also provides access to the complete
+transcripts of live chat sessions, which open in the *Discuss* app.
+
+.. image:: reports/sessions-history.png
+   :align: center
+   :alt: Example of the Sessions History report from the Live Chat application.
+
+The information in this report can be exported or inserted into a spreadsheet. Click the
+:guilabel:`⚙️ (gear)` icon at the end of the words :guilabel:`History` in the top-left corner. This
+reveals a drop-down menu. Click :guilabel:`Export All` to export all sessions to a spreadsheet, or
+:guilabel:`Spreadsheet` to insert the information in a new or existing spreadsheet.
+
+To export only select sessions, first select the sessions to be exported from the list. Then, click
+the checkbox to the left of each individual session to select it. Click the :guilabel:`⚙️ (gear)
+Actions` icon in the top-center of the page, and click :guilabel:`Export`, or
+:guilabel:`Spreadsheet`.
+
+To view the transcript of an individual conversation, select it from the list and click. The page
+conversation opens in the *Discuss* app.
```

Comment:
I see "Live Chat" in the top left corner, so it looks like I am still in the live chat app. Is it just puling the data from discuss and it loads in the live chat app? Might want to clarify that.

I also assume the dicsuss app needs t be installed to view this?

Reply:
The discuss app is always installed (even on one-app free dbs) so I do not think that needs to be specified. 
And yes, *technically* you are still in the Live Chat app, however it is in the Discuss interface,. :+1: